app/controllers/api/v2/access_requests_controller.rb

```diff
@@ -0,0 +1,19 @@
+module API
+  module V2
+    class AccessRequestsController < API::V2::ApplicationController
+      before_action :build_access_request
+
+      def approve
+        result = AccessRequestApprovalService.call(@access_request)
+
+        render status: 200, json: { result: result }
```

Comment:
`render jsonapi: result` maybe

Reply:
Going to follow this: https://discuss.jsonapi.org/t/how-to-you-send-a-success-message/280

Reply:
cool, 

fyi 
> if the request does not exist it returns not found

https://trello.com/c/AaW5wsnh/1362-add-endpoint-to-approve-user-access

Reply:
Rails should do that for us